apps/src/feature_matching.cpp

```diff
@@ -240,18 +241,20 @@ void ICCVTutorial<FeatureType>::detectKeypoints (const typename pcl::PointCloud<
 template<typename FeatureType>
 void ICCVTutorial<FeatureType>::extractDescriptors (typename pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr input, const typename pcl::PointCloud<pcl::PointXYZI>::Ptr& keypoints, typename pcl::PointCloud<FeatureType>::Ptr features)
 {
+  using pcl::dynamic_pointer_cast;  // force ADL
```

Comment:
Is this still needed? `feature_extractor_` is based on a PCL type's `::Ptr` which means using `pcl::dynamic_pointer_cast` in L251 is sufficient

Reply:
Doesn't compile locally without it since the app itself is in the global namespace. You can see the CI for 02277d7 complaining about a `tutorials` file I missed. Unless you want a global `using`...

Reply:
I had evidently not read what you wrote here.